src/scroll-animation.js

```diff
@@ -49,6 +48,9 @@ export default class ScrollAnimation extends Component {
         clearTimeout(tid);
       })
     }
+    if (this.props.animateOnce && this.state.lastVisibility.completely) {
+      return;
```

Comment:
Cool, thanks @hawkins! I forked to debug that querk, ended up making it work but your fix looks much better/simpler.